integration-tests/go.mod

```diff
@@ -25,11 +25,11 @@ require (
 	github.com/slack-go/slack v0.12.2
 	github.com/smartcontractkit/chainlink-automation v1.0.3
 	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240415164156-8872a8f311cb
-	github.com/smartcontractkit/chainlink-testing-framework v1.28.2
+	github.com/smartcontractkit/chainlink-testing-framework v1.28.3-0.20240416135407-16f202719d59
```

Comment:
Will need to do a full release I think.

Reply:
@kalverra pushed a commit that uses a release